portfolio.css

```diff
@@ -0,0 +1,216 @@
+*{
```

Comment:
formatting wise please can you update this file to have a space between your css selector and the `{`

Reply:
eg: `* {`